src/Common/tests/Tests/System/StringTests.cs

```diff
@@ -4800,7 +6048,7 @@ public static void StartEndNegativeTest()
         }
 
         [Fact]
-        public static unsafe void Ctor_SByte()
+        public static unsafe void ConstructorsTest()
```

Comment:
Why rename this? I would keep the sbyte suffix.

Reply:
ugh...because i merged "in the night" :sleeping: